mscore/articulationprop.cpp

```diff
@@ -59,16 +59,16 @@ ArticulationProperties::ArticulationProperties(Articulation* na, QWidget* parent
                   if (a->name.isEmpty() || a->name == "normal")
                         channelList->addItem(tr("normal"));
                   else
-                        channelList->addItem(a->name);
+                        channelList->addItem(qApp->translate("InstrumentsXML", a->name.toUtf8().data()));
                   }
             foreach(const NamedEventList& el, instrument->midiActions()) {
-                  midiActionList->addItem(el.name);
```

Comment:
MIDIAction are not used currrently in instrumentlist. But we do some comparison like https://github.com/musescore/MuseScore/blob/9341f4ea4f0c91605e78b6393f69df0986e22157/libmscore/instrument.cpp#L397 that will need to be tested.